### DISCUSSION TITLE
Does this package work with Azure OpenAI?

### DISCUSSION THREAD

**Discussion by ghost** [srcnalt/OpenAI-Unity#76] [Q&A]
Does this package work with Azure OpenAI? or is it possible to make it work myself?

**Comment by srcnalt** [comment]
Hi @tlspacereu, I do not have access to Azure so I can't really say. Could you guide me here how it looks (docs etc) so I can better understand how we could connect both.

**Reply by ghost** [reply]
Hi@srcnalt , Thank you for the reply! My Azure OpenAI API contains three elements: resource name, deployment name, and API key. Here is a documentary about that: https://learn.microsoft.com/en-us/azure/ai-services/openai/reference

Additionally, I also found another package that does support Azure OpenAI: https://github.com/RageAgainstThePixel/OpenAI-DotNet#azure-openai, I can't understand this document, but maybe it can be helpful.
